src/index.js

```diff
@@ -0,0 +1,25 @@
+import './css/style.css';
+
+import TaskList from './js/taskList.js';
+
+const tasks = new TaskList();
+
+const renderTasks = () => {
+  const docRange = document.createRange();
+  const listContainer = document.getElementById('todo-list-container');
+  const taskList = tasks.getList();
+  let template = '';
+  taskList.forEach((item) => {
+    template += `<li class="todo-list-items">
+            <div class="list-item-data">
+                <input type="checkbox" id="task${item.index}" name="checkbox" value="${item.index}">
+                <label for="task${item.index}">Task ${item.index + 1}</label>
+            </div>
+            <div class="list-item-data">
+                <i id="delete${item.index}" class="fas fa-trash-alt"></i>
+            </div>
+        </li>`;
+  });
```

Comment:
- [ ] I suggest that you add your simple to-do list in an array(array of objects) and each task should have 3 keys `description`, `completed`, and `index` instead of adding them like HTML elements

Reply:
- [ ] Please work on the function that iterates over the `tasks` array and populates the HTML list item element for each task, `tasks` should be rendered dynamically to the DOM, and the `tasks` should appear in order of the `index` values for each task.

Reply:
> * [ ]  I suggest that you add your simple to-do list in an array(array of objects) and each task should have 3 keys `description`, `completed`, and `index` instead of adding them like HTML elements

The **Array** is the ``` taskList ``` getting generated using ```task.js``` default object having the keys ```index ```, ```completed```, ```description```. Please consider checking the code flow. I have followed proper ```OOP``` with JS modular architecture.

Reply:
> * [ ]  Please work on the function that iterates over the `tasks` array and populates the HTML list item element for each task, `tasks` should be rendered dynamically to the DOM, and the `tasks` should appear in order of the `index` values for each task.

Hi @DeliceLydia ,The ``` forEach``` loop on ```taskList``` array is rendering everything dynamically on HTML DOM, in proper order of index starting from ```0``` index till ```2```, as I am rendering 3 elements. Please consider rechecking the code follow. What you have requested for is already implemented.

Reply:
Hello @RitobrotoMukherjee, Thank you for reaching out! I get what you mean only just it was suggested for this milestone to add manually some simple tasks in the array then render them dynamically to the DOM just like:
```
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];

```
Ask another review and it will get approved you can add extra information while submitting